vllm/platforms/__init__.py

```diff
@@ -2,25 +2,28 @@
 
 import torch
 
-from vllm.utils import is_hpu, is_tpu
-
 from .interface import Platform, PlatformEnum, UnspecifiedPlatform
 
 current_platform: Optional[Platform]
 
+is_hpu = False
+try:
+    import os
+    from importlib import util
+    is_hpu = util.find_spec('habana_frameworks') is not None
+
+except Exception:
+    pass
+
 if torch.version.cuda is not None:
     from .cuda import CudaPlatform
     current_platform = CudaPlatform()
 elif torch.version.hip is not None:
     from .rocm import RocmPlatform
     current_platform = RocmPlatform()
-elif is_tpu():
```

Comment:
Restore is_tpu() method. Previous review was about not needing to introduce our check for tpu, but we cant remove existing ones.

Reply:
done